src/main/java/controllers/Controller.java

```diff
@@ -194,7 +194,7 @@ public List<ClassInfo> getAllClasses() {
             // Connect to database and get fitness stats data
             Connection connection = DriverManager.getConnection(databaseURL, databaseUsername, databasePassword);
             Statement statement = connection.createStatement();
-            statement.executeQuery("SELECT * FROM timeslots NATURAL JOIN classes NATURAL JOIN participants");
+            statement.executeQuery("SELECT * FROM timeslots NATURAL JOIN classes LEFT JOIN participants ON classes.class_id=participants.class_id");
```

Comment:
```suggestion
            statement.executeQuery("SELECT * FROM timeslots NATURAL JOIN classes");
```
I didn't actually mean to include participants here. Came from when I was making a method to get all of one members classes.

Reply:
Fixed using your recommendation: e8d6079